packages/azure-functions/package.json

```diff
@@ -29,7 +29,9 @@
     "@defra/bng-geoprocessing-service": "1.85.0",
     "@defra/bng-metric-service": "1.85.0",
     "@defra/bng-utils-lib": "1.85.0",
-    "moment": "^2.29.4"
+    "moment": "^2.29.4",
+    "ip": ">=2.0.1"
```

Comment:
why is this being added?

Reply:
I think ip is a dependency of lerna and so instead perhaps it is lerna that needs upgrading?